platform/platform-api/src/main/java/org/codice/ddf/platform/services/common/Describable.java

```diff
@@ -26,7 +28,7 @@
      *
      * @return the version of the item being described (example: 1.0)
      */
-    String getVersion();
+    @NotNull String getVersion();
```

Comment:
I do not like seeing these `@NotNull` annotations all over. Starting a thread in the ddf-developers to discuss this.